<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1464</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schulfreund</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ente.png</normaloff>ente.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_kal">
     <attribute name="title">
      <string>Kalender</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Stunden des Tages&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView_fokus">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>240</height>
       </rect>
      </property>
      <property name="gridVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView_tag">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>270</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Niklas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;08.03.23. Klausur EF&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wichtige Ereignisse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Achte heute auf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView_ereignisse">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>391</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addLessonButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <zorder>listView_tag</zorder>
     <zorder>listView_ereignisse</zorder>
     <zorder>label</zorder>
     <zorder>listView_fokus</zorder>
     <zorder>calendarWidget</zorder>
     <zorder>label_5</zorder>
     <zorder>label_3</zorder>
     <zorder>label_2</zorder>
     <zorder>label_4</zorder>
     <zorder>label_6</zorder>
     <zorder>addLessonButton</zorder>
    </widget>
    <widget class="QWidget" name="tab_kurse">
     <attribute name="title">
      <string>Kurse</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_sus">
     <attribute name="title">
      <string>Sch체ler*innen</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_sequ">
     <attribute name="title">
      <string>Sequenzen</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_faecher">
     <attribute name="title">
      <string>F채cher</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_sett">
      <attribute name="title">
      <string>Einstellungen</string>
      </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1464</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuKalender">
    <property name="title">
     <string>Kalender</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSch_ler">
    <property name="title">
     <string>Sch&amp;체ler</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuKurse">
    <property name="title">
     <string>Kurse</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSequenzen">
    <property name="title">
     <string>Se&amp;quenzen</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuF_cher">
    <property name="title">
     <string>F&amp;채cher</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuKalender"/>
   <addaction name="menuKurse"/>
   <addaction name="menuSch_ler"/>
   <addaction name="menuSequenzen"/>
   <addaction name="menuF_cher"/>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mockup.qrc"/>
 </resources>
 <connections/>
</ui>
